src/core/types.ts

```diff
@@ -96,5 +101,60 @@ export interface CGIData {
   [x: string]: unknown
 }
 
+export type ProcessStats = {
+ load: {
+   one: number,
+   five: number,
+   fifteen: number
+ };
+ mem: {
+   total?: number,
+   free?: number,
+   buffers?: number,
+   cached?: number,
+   free_total?: number
+ }
+}
+
+export type TransportOptions = {
+  method: 'GET' | 'POST',
+  headers?: Record<string, number | string | string[] | undefined>,
+  endpoint: string,
+  maxObjectDepth?: number,
+  logger: Logger
+  async?: boolean // don't like this here because it's only for browser
```

Comment:
Lool. I wonder if there are any users that actually want their reporting to block the thread. There probably are.🤔

Reply:
🤷